recipes/ga4ghmongo/meta.yaml

```diff
@@ -14,7 +14,6 @@ requirements:
   build:
     - python
     - setuptools
-    - mongoengine
```

Comment:
Wow, you're fast :smile:

Reply:
👍 Have two more packages to go after this one.